code/modules/antagonists/brother/brother.dm

```diff
@@ -31,22 +31,30 @@
 	owner.special_role = null
 	return ..()
 
-/datum/antagonist/brother/proc/give_meeting_area()
-	if(!owner.current || !team || !team.meeting_area)
-		return
-	to_chat(owner.current, "<B>Your designated meeting area:</B> [team.meeting_area]")
-	antag_memory += "<b>Meeting Area</b>: [team.meeting_area]<br>"
+/datum/antagonist/brother/antag_panel_data()
+	var/brother_text = get_brother_names()
```

Comment:
Don't need this var here.

Reply:
yo sorry I did see this, why is this unnecessary

Reply:
You can just inline it if you only use it once.
`"Conspirators : [get_brother_names()]]`